doc/nrf/libraries/others/identity_key.rst

```diff
@@ -0,0 +1,41 @@
+.. _lib_identity_key:
+
+Identity key
+############
+
+.. contents::
+   :local:
+   :depth: 2
+
+The identity key library manages an asymmetric key used for identity services on devices with the Arm CryptoCell and KMU peripherals.
+
+Functionality
+*************
+
+This library manages the identity keys, which are asymmetric keys intended to provide identity to specific devices.
```

Comment:
TF-M uses the word "attestation key" for this key unless I have misunderstood something.

Should we use the same terminology to reduce the number of words we use for the same thing?

Reply:
We've used "Identity key" too many places already.

It's too late to fix this usability issue.